hack/local-up-cluster.sh

```diff
@@ -39,6 +39,8 @@ WAIT_FOR_URL_API_SERVER=${WAIT_FOR_URL_API_SERVER:-10}
 ENABLE_DAEMON=${ENABLE_DAEMON:-false}
 HOSTNAME_OVERRIDE=${HOSTNAME_OVERRIDE:-"127.0.0.1"}
 CLOUD_PROVIDER=${CLOUD_PROVIDER:-""}
+ETCD_HOST=${ETCD_HOST:-"127.0.0.1"}
```

Comment:
it's a little convoluted, but `hack/local-cluster-up.sh` [sources `hack/lib/init.sh`](https://github.com/kubernetes/kubernetes/blob/master/hack/local-up-cluster.sh#L50), which in turn [sources `hack/lib/etcd.sh`](https://github.com/kubernetes/kubernetes/blob/master/hack/lib/init.sh#L36).